.gitignore

```diff
@@ -5,3 +5,4 @@ bin/terraform*
 platform-tests/pkg
 platform-tests/src/results
 terraform/.terraform/
+.venv/
```

Comment:
This *feels* like a non-repo-specific change that should live in your per-user .gitignore.

Reply:
TIL per-user gitignore files are a thing...I will update!